### PR TITLE
ultilize gem shoulda-matchers for testing subject model

### PR DESCRIPTION
**Pull request by nguyenthanhtu95**
image

### CODE REVIEW

**Comment by nguyenthanhtu95** [comment]
ready

File: spec/models/subject_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RSpec.describe Subject, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of(:title) }
+    it { should validate_length_of(:title).is_at_most(255) }
+    it { should validate_presence_of(:description) }
+    it { should validate_length_of(:description).is_at_most(1000) }
+    it { should belong_to(:creator) }
+    it { should have_many(:tasks) }
+    it { should have_many(:course_subjects) }
+    it { should delegate_method(:name).to(:creator) }
```
Comment (phamvanbk11) [inline spec/models/subject_spec.rb]: Còn thiếu 2 cái scope

Comment (nguyenthanhtu95) [reply spec/models/subject_spec.rb]: added

**Comment by nguyenthanhtu95** [comment]
ready

**Pull request merged**